.github/workflows/build.yaml

```
@@ -171,6 +171,15 @@ jobs:
             -e `pwd`/../AtomVM/build/libs/eavmlib/src/eavmlib.avm \
             -e `pwd`/../AtomVM/build/libs/estdlib/src/estdlib.avm
 
+    - name: Upload sounds.bin
+      if: matrix.variant == "4mb"
```

actionlint (1.7.10)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)